packages/mock-app/src/components/Router/Router.tsx

```diff
@@ -40,7 +40,7 @@ function Router() {
       <Route path='*' element={<Navigate to='/404' />} />,
       {/* Iterate through the appConfig to dynamically generate routes */}
       {appConfig.apps.map(renderRoutes)}
-      {appConfig.generateFeatures.map(renderRoutes)}
+      {appConfig.generalFeatures.map(renderRoutes)}
```

Comment:
Can you provide the latest version of `app-config.json` in the PR description? I noticed the `generateFeatures` field name has been updated to `generalFeatures`.